pkg/goDB/keyval.go

```diff
@@ -68,9 +71,9 @@ func (k Key) MarshalJSON() ([]byte, error) {
 			Dport uint16 `json:"dport"`
 			Proto string `json:"ip_protocol"`
 		}{
-			RawIPToString(k.Sip[:]),
-			RawIPToString(k.Dip[:]),
-			uint16(uint16(k.Dport[0])<<8 | uint16(k.Dport[1])),
```

Comment:
The outer uint16() should be unnecessary (doesn't your linter make mimimi about that?)